src/client/al-api-client.ts

```diff
@@ -632,7 +632,10 @@ export class AlApiClient
 
   protected async calculateRequestURL( params: APIRequestParams ):Promise<string> {
     let fullPath:string = null;
-    if ( params.service_name && params.service_stack === AlLocation.InsightAPI && ! params.noEndpointsResolution ) {
+    if ( params.service_name
+      && (params.service_stack === AlLocation.InsightAPI
```

Comment:
Please, consistent nesting formatting is really important here --

```
if ( params.service_name
      && (params.service_stack === AlLocation.InsightAPI || params.service_stack === AlLocation.IntegrationsAPI )
      && ! params.noEndpointsResolution ) {
```

That logic is hard to read properly flat :)